docs/tempo/website/grafana-agent/_index.md

```diff
@@ -107,7 +107,17 @@ For a detailed description, go to [Tail-based sampling](tail-based-sampling).
 
 The Agent can take advantage of the span data flowing through the pipeline to generate Prometheus metrics.
 
-Go to [Span metrics](span-metrics) for a more detailed explanation of the feature.
+Go to [Span metrics](../server_side_metrics/span_metrics) for a more detailed explanation of the feature.
+
+#### Service graphs
+
+Service graphs are the representations of the relationships between services within a distributed system.
+
+This service graphs processor builds a map of services by analysing traces, with the objective to find _egdes_.
+Edges are spans with a parent-child relationship, that represent a jump (e.g. a request) between two services.
+The amount of request and their duration are recorded as metrics, which are used to represent the graph.
+
+To read more about this processor, go to its [section](service_graphs.md)
```

Comment:
unsure about these links. I think it might be `[section](./service-graphs)`. You can test by going to running `make docs-test`. It will locally host a rendered version of our docs.